### DISCUSSION TITLE
Handle Large / Big Data from APi with Datatable

### DISCUSSION THREAD

**Discussion by Zen225** [primefaces/community#824] [PrimeReact] [answered]
Halo

I just makes some Module CRUD with React JS and Use Sakai PrimeReact for the Theme

How to do handle Big / Large Datasets using Datatable form Sakai Prime React ( Next JS ) ? any Suggestion ?

Thanks
Best Regrads

**Comment by tugcekucukoglu** [comment]
Moving PrimeReact channel.

**Reply by melloware** [reply, marked as answer]
@Zen225 here is a project using PrimeReact 10.2 that shows using all features of Lazy Datatable with real REST Services.

https://github.com/melloware/quarkus-primereact

Just run `mvn quarkus:dev` to see it all in action.
